tests/common.py

```diff
@@ -1,17 +1,33 @@
 import hashlib
 
-from data_diff.database import CHECKSUM_HEXDIGITS, MD5_HEXDIGITS
+from data_diff import database as db
 import logging
 
 logging.basicConfig(level=logging.WARN)
 
-TEST_MYSQL_CONN_STRING = "mysql://mysql:Password1@localhost/mysql"
+TEST_MYSQL_CONN_STRING: str = "mysql://mysql:Password1@localhost/mysql"
+TEST_POSTGRES_CONN_STRING: str = None
```

Comment:
We should be able to hardcode this one too:

```
postgres://postgres:Password1@localhost/postgres
```

Reply:
Sure